decidim-admin/spec/features/admin_manages_user_groups_spec.rb

```diff
@@ -5,18 +5,25 @@
 
 describe "Admin manage user groups", type: :feature do
   include_context "participatory process admin"
-  let!(:user_groups) { create_list(:user_group_membership, 10, user: create(:user, organization: organization)) }
+  let!(:user_groups) { create_list(:user_group_membership, 10, user: create(:user, organization: organization)).map(&:user_group) }
```

Comment:
This test was broken before and I found the shortest path to fix it. I was afraid the context might be using the variable so I didn't want to change it. Let's have a look.